### ISSUE TITLE
LeaderBoard evaluator object has no attribute manager

### ISSUE THREAD

**Issue by waleedrafique51** [opendilab/InterFuser#85]
The work present by author is excellent. I need help how to resolve this issue. Can author help me to resolve this issue for data collection .Here is error message  .please help me to sort out this bug
![error](https://github.com/opendilab/InterFuser/assets/66169176/b8c6f90f-4036-4629-88ea-de040e61ea1b)

**Comment by deepcs233** [comment]
Hi!
Sorry for the late reply. Could you check whether the Carla simulator ran normally? Or can you restart the simulator.

**Comment by waleedrafique51** [comment]
Yes my carla is working properly,The issue is due to leaderboard script in learderboardevaluator.py.I have modified some lines of codes with the new leaderboard script .Now it is working

**Comment by rogerfinnerty** [comment]
@waleedrafique51 I am getting the same issue, how did you fix this?

**Comment by Ruyi001** [comment]
@waleedrafique51 I am getting the same issue, how did you fix this?
